src/init.jl

```diff
@@ -87,7 +88,8 @@ function init(args)
     raw_input[] = Socket(ROUTER)
     requests[] = Socket(ROUTER)
     control[] = Socket(ROUTER)
-    heartbeat[] = Socket(ROUTER)
+    heartbeat_context[] = Context()
+    heartbeat = Socket(heartbeat_context[], ROUTER)
```

Comment:
@halleysfifthinc, should this be indexing like `heartbeat[] = ...` since `heartbeat` is still a Ref?

Reply:
D'oh, bad rebase on my part. Yes that should be indexed.